un_sdg/__init__.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+#from standard_importer.import_dataset import USER_ID
+# Dataset constants.
+DATASET_NAME = "United Nations Sustainable Development Goals Indicators"
+DATASET_AUTHORS = "United Nations"
+DATASET_VERSION = "2021-03"
+DATASET_LINK = "https://unstats.un.org/sdgs/indicators/database/" # Have to request dataset by email
+DATASET_RETRIEVED_DATE = "10-May-2021"
```

Comment:
```suggestion
```

Unused.

Reply:
Now used in create_sources, updated [here](https://github.com/owid/importers/commit/906998fa0d4c1cf0c538dc3f6c6f802f166092a6#diff-2f6e417c9a3497a8224254ef660c6fe8c3a6f55fc2f5df33c3b5b63a12658a08) to be the date the raw data file was last manipulated, according to `os.stat()`